docs/PRINCIPLES.md

```diff
@@ -6,9 +6,7 @@ Spice.ai is built upon a foundation of first principles.
 
 - Developer experience first
 - Bring data and ML to your application
-- API first
-- Composable from community driven-components
```

Comment:
Composable should probably also remain. We still have Datasets, Models, and will later have Functions.